Library/Formula/kubernetes-cli.rb

```diff
@@ -15,11 +15,12 @@ class KubernetesCli < Formula
   depends_on "go" => :build
 
   def install
+    os = if OS.linux? then 'linux' else 'darwin' end
```

Comment:
Please use `?` and `:` and double quotes, like the line below. See the output of `brew audit --strict kubernetes-cli`